project/src/main/java/com/google/sps/utilities/Constants.java

```diff
@@ -0,0 +1,6 @@
+package com.google.sps.utilities;
```

Comment:
Maybe we can call this class  DatabaseConstants?

Reply:
Done